Interop 2023 post (#9479)
GoogleChrome/web.dev: src/site/content/en/blog/interop-2023/index.md

---
layout: post

> title: "Interop 2023: continuing improving the web for developers"

Reviewer: ```suggestion
title: "Interop 2023: continuing to improve the web for developers"
```

subhead:
    "Again in 2023, all major browser vendors, and other stakeholders, work together to solve the top browsers compatibility issues."
description:
    "Learn more about how all browser vendors, and other stakeholders, have come together to solve the top browsers compatibility issues identified by web developers. Interop 2023 will improve the experience of developing for the web across a number of key areas."
authors:
  - robertnyman
  - foolip
  - rachelandrew
hero: "image/RYmV5NPuMZRoF3PVwIXTUpdYeQ23/NqhNbZwtnkSEgDtlXdHa.jpg"

> alt: "A picture of a rounded stairway"

Reviewer: ```suggestion
alt: "A picture of a curved stairway."
```

---
layout: post
title: "Interop 2023: continuing improving the web for developers"
subhead:
    "Again in 2023, all major browser vendors, and other stakeholders, work together to solve the top browsers compatibility issues."
description:
    "Learn more about how all browser vendors, and other stakeholders, have come together to solve the top browsers compatibility issues identified by web developers. Interop 2023 will improve the experience of developing for the web across a number of key areas."
authors:
  - robertnyman
  - foolip
  - rachelandrew
hero: "image/RYmV5NPuMZRoF3PVwIXTUpdYeQ23/NqhNbZwtnkSEgDtlXdHa.jpg"
alt: "A picture of a rounded stairway"
date: 2023-02-01

> updated: 2023-01-31

Reviewer: ```suggestion
```
